zenpacklib.py

```diff
@@ -621,9 +624,21 @@ def _create_indexes(cls, zcatalog, spec):
                 # reindex all objects of this type so they are added to the
                 # catalog.
                 results = ICatalogTool(context).search(types=(classname,))
+
```

Comment:
I added the guards from https://github.com/zenoss/zenpacklib/pull/150 but I'm not sure if I should have just removed the guards on my side and left them in the other PR?